src/plugins/intel_cpu/tests/functional/single_layer_tests/instances/arm/eltwise.cpp

```diff
@@ -0,0 +1,108 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "single_layer_tests/classes/eltwise.hpp"
+#include "shared_test_classes/single_layer/eltwise.hpp"
+#include "test_utils/cpu_test_utils.hpp"
+#include "test_utils/fusing_test_utils.hpp"
+
+using namespace InferenceEngine;
+using namespace CPUTestUtils;
+using namespace ngraph::helpers;
+using namespace ov::test;
+
+namespace CPULayerTestsDefinitions {
+namespace Eltwise {
+namespace {
+static const std::vector<InputShape> bitwise_in_shapes_4D = {
+    {
+        {1, -1, -1, -1},
+        {
+            {1, 64, 4, 4},
+            {1, 64, 1, 1}
+        }
+    },
+    {{1, 64, 4, 4}, {{1, 64, 4, 4}}}
+};
+
+const auto params_4D_bitwise = ::testing::Combine(
+    ::testing::Combine(
+        ::testing::Values(bitwise_in_shapes_4D),
+        ::testing::ValuesIn({
+            ngraph::helpers::EltwiseTypes::BITWISE_AND,
+            ngraph::helpers::EltwiseTypes::BITWISE_OR,
+            ngraph::helpers::EltwiseTypes::BITWISE_XOR
+        }),
+        ::testing::ValuesIn(secondaryInputTypes()),
+        ::testing::ValuesIn({ ov::test::utils::OpType::VECTOR }),
+        ::testing::ValuesIn({ ov::element::Type_t::i8, ov::element::Type_t::u8, ov::element::Type_t::i32 }),
+        ::testing::Values(ov::element::Type_t::undefined),
+        ::testing::Values(ov::element::Type_t::undefined),
+        ::testing::Values(ov::test::utils::DEVICE_CPU),
+        ::testing::Values(ov::AnyMap())),
+    ::testing::Values(CPUSpecificParams({ nhwc, nhwc }, { nhwc }, {}, "ref")),
```

Comment:
The thing I missed last time. Why only `nhwc` layout is tested? I would expect `nchw` to be covered as well.

Reply:
added, thanks!